### PR TITLE
Address checksum for RSK chains. RSKIP60

### PR DESCRIPTION
**Pull request by vbaranov**
Address checksum in RSK chains differs from Ethereum chains specification. In Ethereum chains it is standardized with [EIP-155] (https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md). Whereas it follows [RSKIP60](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP60) for RSK.

This PR implements RSKIP60.

Address checksum example in different chains:
0xDf682CB153E8af3E13E90e9ACD79a39DDe98eBd8 (ETH Mainnet)
0xdF682cb153E8AF3E13E90e9aCD79A39Dde98EBD8 (RSK Mainnet)
0xdf682cb153e8af3e13e90E9aCD79A39DDe98eBd8 (RSK Testnet)

### CODE REVIEW

**Comment by alepc253** [comment]
@vbaranov Looks good! However, I'm not sure if it is considering input validation. Does it validate correctly an RSK checksummed address entered as "Recipient Address" in "Send Transaction"?
This is what I mean: https://github.com/ethereum/web3.js/blob/2.x/packages/web3-utils/src/Utils.js#L103

**Comment by fedejinich** [comment]
@vbaranov I've been testing this new feature, and I've found two unexpected behaviours. The first one, as previous mentioned @alepc253, is that you can't send any transaction (tokens and rbtc) to an RSK  Mainnet/Testnet direction, I guess its because of checksum validation. The other issue is at "Show QR code" section, the displayed checksum doesn't match the RSK checksum (still beeing an ethereum checksum). 

First issue
image

Second issue
image


Anyways, great job and hope to be merged to master!

**Comment by vbaranov** [comment]
@alepc253 @fedejinich thank you for the comments. Indeed, eth checksum was used for address validation in pending tx. It has been fixed https://github.com/poanetwork/nifty-wallet/pull/294/commits/c1142ddda9683e454b3cb5dfac229046de7ed52e.

As for the 2nd issue, we displayed not address checksum, but the raw address in the QR code screen. I've added checksum for displayed address and to `copy` action https://github.com/poanetwork/nifty-wallet/pull/294/commits/d973c28d2d0b38b108df57e9fc9adc4abde85364. What is displayed behind the QR-code is defined by [EIP-831](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-831.md). What should we show behind QR-code for RSK-chains? Is any RSKIP for that?

**Comment by alepc253** [comment]
@vbaranov we don't have any RSKIP for that. However, I suggest two options (according to other tools that integrate us) to show behind QR code, order by preference:
1) Only the address checksummed (no name prefix)
2) Only the address in lowercase (no name prefix)
Thanks!

**Comment by vbaranov** [comment]
@alepc253 the 1st option has been implemented.

@patitonar this PR is ready for review

File: old-ui/app/util.js [hunk 1]
```diff
@@ -375,3 +380,37 @@ function getAllKeyRingsAccounts (keyrings, network) {
...
+function ifRSK (network) {
+  if (!network) return false
+  const numericNet = isNaN(network) ? network : parseInt(network)
+  return numericNet === RSK_CODE || numericNet === RSK_TESTNET_CODE
+}
+
+function toChecksumAddressRSK (address, chainId = null) {
+  const zeroX = '0x'
+  const stripAddress = ethUtil.stripHexPrefix(address).toLowerCase()
+  const prefix = chainId != null ? (chainId.toString() + zeroX) : ''
```
Comment (patitonar) [inline old-ui/app/util.js]:
```suggestion
  const prefix = chainId !== null ? (chainId.toString() + zeroX) : ''
```

**Pull request merged**